lib/helper.js

```diff
@@ -44,7 +45,7 @@ var validationType = function(value, validation, key){
       return {value: value};
     }else{
       if(types[validation].type) validation = types[validation].type;
-      return {error: key + ': ' + value + ', has to be ' + validation + ' type.', param: key};
+      return {error: key + ': ' + escapeHtml(value) + ', has to be ' + validation + ' type.', param: key};
```

Comment:
I looked into this further and now I understand that https://github.com/websdotcom/sails-hook-validator/blob/master/lib/errorsParser.js#L40 is sending the response with the reflected XSS.

Can we just add an html escape on the final output? It seems like that's a better place for it since it covers all possible inputs and avenues for XSS, rather than just this one.

I'm not sure why the validator returns text/html in this format but I guess it doesn't really matter since we don't expect users to ever encounter this.